### PR TITLE
Add feature flag for QuickAddBundleView

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Put QuickAdd with the bundle view under a feature flag
- Add new QuickAddBundleView component and story for easy toggle of components

I added back the original `QuickAdd` so it might look like a lot of changes, but that component is the same as current implementation.

### Switched off
Screenshot 2023-12-13 at 10 39 49

### Switched on
Screenshot 2023-12-13 at 10 44 11

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3715** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3715?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3716** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3716?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3717** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3717?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3723** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3723?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3726** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3726?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3727** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3727?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #3730** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3730?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3730?utm_source=stack-comment).

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3715** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3715?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3716** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3716?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3717** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3717?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3723** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3723?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3726** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3726?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3727** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3727?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #3730** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3730?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3730?utm_source=stack-comment).

File: apps/store/src/components/QuickAdd/AddToCartButton.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { datadogLogs } from '@datadog/browser-logs'
 import { datadogRum } from '@datadog/browser-rum'
-import { useTranslation } from 'next-i18next'
+import { ReactNode } from 'react'
```
Comment (robinandeer) [inline apps/store/src/components/QuickAdd/AddToCartButton.tsx:3]: nit: type

**Review by robinandeer — COMMENTED** [review]
is this rebased on top of other changes? just so many changes...

**Comment by gustaveen** [comment]
> is this rebased on top of other changes? just so many changes...

Nope, to keep it simple I added back the original `QuickAdd` component and put the changes in another component and story, so that's why it looks like a lot :) Didn't want to have multiple conditionals in the `QuickAdd` component @robinandeer

**Review by robinandeer — APPROVED** [review]
Okey now I understand - don't have much to add - looks great!

**Comment by gustaveen** [comment]
### Merge activity

* **Dec 14, 4:58 AM**: @@gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3730).

**Pull request merged**